From 4d3d9e190ab0fc3558b356e6033057549262cf33 Mon Sep 17 00:00:00 2001
From: eroen <eroen@occam.eroen.eu>
Date: Sat, 12 Jan 2013 02:17:37 +0100
Subject: [PATCH] lxc-0.8.0-r1: Make man page generation optional

Generating the man pages depends on docbook.
There is a precedent for this (also docbook related) in
libcmis/libcmis-0.2.3-r1
---
 app-emulation/lxc/Manifest            | 4 ++--
 app-emulation/lxc/lxc-0.8.0-r1.ebuild | 8 ++++----
 app-emulation/lxc/metadata.xml        | 3 +++
 3 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/app-emulation/lxc/Manifest b/app-emulation/lxc/Manifest
index 193b493..4dd9865 100644
--- a/app-emulation/lxc/Manifest
+++ b/app-emulation/lxc/Manifest
@@ -1,5 +1,5 @@
 AUX lxc.initd.2 3673 SHA256 bb7780168695e934549909c5c7baa7d6cdc92b0e19093a3d04a1313081a832e1 SHA512 25b0474d6f16fcede89a8e17518a658f810cb554eaef10aae3ce1fdd8c8640118aa6375cbee443f0b26e8086eb7c25375ceaaaf29a4110bd779f94b2f4b0d300 WHIRLPOOL 28f56d71b95b26c7f6ad5402a37413a973bc7bdf08fd3963ad192888ae1b49809a56c16009bc6049061d419ecffb1279e153e6ef5acbe06956efca154eae6a01
 DIST lxc-0.8.0-backports-1.tar.xz 2424 SHA256 5b68b96b9fbc2d0f5b2ad2e7fb996a3d837114237a306de39d9939f31756a522 SHA512 9898d78836cc546e80015a18070e3976118d2efb006584ecfb2887a4474598766e023eb6c120c16dae2e94dddc33c8a4e928fd44bc6fb2260a1913c06f5d374e WHIRLPOOL ef11f25ec8891f431be7aed32b38c0cc51e1096a37ea71c38aed4448ab0305784b510e2b55bdc73e5ef2128344a6c88adb3add798323f77e276b5185e53f5ef9
 DIST lxc-0.8.0.tar.gz 309973 SHA256 eac833730b94e423d4bef6834bc4a716437d2abd6ab8b24334c506aaaa08692c SHA512 47333e2010d0f4488b8876ba933f2d26304874bec0371a7ab3b6e5bd1a50a7d2312ff8507e273f1ee2341a7f2e3b79cd71e5e19ac31006a4e429ee96b01733f6 WHIRLPOOL 693b7d354dec7c6102d5ce3a42a9b618e213fd9ee8c592068ebadb7f25094606abce6c3da77b483fab0938410215010f71d487f336856c87327d6a98f20b65dd
-EBUILD lxc-0.8.0-r1.ebuild 4504 SHA256 2199fc1bce5cabcf082519476fd0b3bed090767c0b72ab607f230b7a1d50da0a SHA512 62074bb7f4f976e544f5bfe3cbaffdc6748982e68ba65bac14c16d64c8bb1520f3976d4016756be9f60d6a5307d0afa8975b27fded4db296c6705eb748c3bc90 WHIRLPOOL 2b64f73490bff91801afe86907c90dfba4a20b0627fa49c3f96bc3315a01527affabbefa64c606b1cb401a66327f361cca7c74155ecb427713b3d98fa85693e4
-MISC metadata.xml 653 SHA256 35f64761067f47ff7abd0e3f3cb57f4bab888a2546bf3267d59987505101a190 SHA512 d227bc432c37518190c224b280116439dde42678f93925ae3a8eb2d2e91918377b33171d8e72ae4ab12f3e5d9faead4223a60e53bb4916d7770982deb64b0b70 WHIRLPOOL 4067036bca1ad2906cdba69c42c0ca5b14042ad7eab2ac8e9a2fbe4eb61e8a3972e4d40b6d9d6e7109a0d2f84a8598a8a9560dbddd43b35633f44ad5362937b0
+EBUILD lxc-0.8.0-r1.ebuild 4526 SHA256 aa53edadc142486ea6e4d257b8f11abd290259367b32a9b42e1ebff9ba75d7e4 SHA512 0e95e6595d45fdd15e7b1f1347a037f2360b26361a8e33ca8f49f56f8f3075dc266dbfce60f3f8bbd2e8261e40490b2f7687783c67fbd2e24994196149c1e1be WHIRLPOOL dd64e2f07cbeffa2af3791fe9c59cb5492f67471885d6a6d278bb64493031aaad1db11e3c8f68fa288ac1c14e00cb8e5824dec376faa1a1eab83db41e68ba58d
+MISC metadata.xml 713 SHA256 437f1639e061cffc3fcb94f4937d2f538cd4679fcc6ba79bc1700bb6a49f6ded SHA512 270e7156e50c690b0dd890e1760fdc6e5d9ce6cd51e7c20a85ef483e73f2c7c1357450c10f3d88547c65c40de227b00297b5d96ce8687fa4770eabf516dfe213 WHIRLPOOL 43cd495d8763575329aa289054b36c07064a8c74a20df81bd5bfdb49a4dd09aa4908fa5e2ab3ce42fb06ed391c708fb6c406b031d8c6bbc58c3591f73a356f3e
diff --git a/app-emulation/lxc/lxc-0.8.0-r1.ebuild b/app-emulation/lxc/lxc-0.8.0-r1.ebuild
index 96dfa31..4416c60 100644
--- a/app-emulation/lxc/lxc-0.8.0-r1.ebuild
+++ b/app-emulation/lxc/lxc-0.8.0-r1.ebuild
@@ -1,4 +1,4 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2013 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Header: /var/cvsroot/gentoo-x86/app-emulation/lxc/lxc-0.8.0-r1.ebuild,v 1.1 2012/11/14 02:15:10 flameeyes Exp $
 
@@ -24,12 +24,12 @@ KEYWORDS="~amd64 ~ppc64 ~x86"
 
 LICENSE="LGPL-3"
 SLOT="0"
-IUSE="examples"
+IUSE="examples man"
 
 RDEPEND="sys-libs/libcap"
 
 DEPEND="${RDEPEND}
-	app-text/docbook-sgml-utils
+	man? ( app-text/docbook-sgml-utils )
 	>=sys-kernel/linux-headers-3.2"
 
 RDEPEND="${RDEPEND}
@@ -99,7 +99,7 @@ src_configure() {
 		--docdir=/usr/share/doc/${PF} \
 		--with-config-path=/etc/lxc	\
 		--with-rootfs-path=/usr/lib/lxc/rootfs \
-		--enable-doc \
+		$(use_enable doc man) \
 		--disable-apparmor \
 		$(use_enable examples)
 }
diff --git a/app-emulation/lxc/metadata.xml b/app-emulation/lxc/metadata.xml
index 694fd72..5ae435b 100644
--- a/app-emulation/lxc/metadata.xml
+++ b/app-emulation/lxc/metadata.xml
@@ -17,5 +17,8 @@
       around as they use LXC in contexts where the init script is not
       useful.
     </flag>
+	<flag name="man">
+	  Build and install man pages.
+	</flag>
   </use>
 </pkgmetadata>
-- 
1.8.1

